[2025-08-16 22:24:37,071] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_business_types manual__2025-08-16T22:24:29.163570+00:00 [queued]>
[2025-08-16 22:24:37,083] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_business_types manual__2025-08-16T22:24:29.163570+00:00 [queued]>
[2025-08-16 22:24:37,084] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 22:24:37,085] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-16 22:24:37,086] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 22:24:37,098] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): load_business_types> on 2025-08-16 22:24:29.163570+00:00
[2025-08-16 22:24:37,108] {standard_task_runner.py:52} INFO - Started process 158 to run task
[2025-08-16 22:24:37,125] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '1_load_initial_data', 'load_business_types', 'manual__2025-08-16T22:24:29.163570+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/initialise_data.py', '--cfg-path', '/tmp/tmpol4hv4q2', '--error-file', '/tmp/tmp9w4o8rms']
[2025-08-16 22:24:37,131] {standard_task_runner.py:80} INFO - Job 23: Subtask load_business_types
[2025-08-16 22:24:37,229] {task_command.py:369} INFO - Running <TaskInstance: 1_load_initial_data.load_business_types manual__2025-08-16T22:24:29.163570+00:00 [running]> on host f4989c8fe60b
[2025-08-16 22:24:37,327] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=1_load_initial_data
AIRFLOW_CTX_TASK_ID=load_business_types
AIRFLOW_CTX_EXECUTION_DATE=2025-08-16T22:24:29.163570+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-16T22:24:29.163570+00:00
[2025-08-16 22:24:37,329] {sql.py:262} INFO - Executing: COPY dbt_raw_data.business_types FROM '/sample_data/business_types_v1.csv' DELIMITER ';' CSV HEADER;
[2025-08-16 22:24:37,347] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 22:24:37,384] {logging_mixin.py:115} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/connection.py:339 DeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
[2025-08-16 22:24:37,410] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 22:24:37,418] {sql.py:375} INFO - Running statement: COPY dbt_raw_data.business_types FROM '/sample_data/business_types_v1.csv' DELIMITER ';' CSV HEADER;, parameters: None
[2025-08-16 22:24:37,426] {sql.py:384} INFO - Rows affected: 9
[2025-08-16 22:24:37,456] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=1_load_initial_data, task_id=load_business_types, execution_date=20250816T222429, start_date=20250816T222437, end_date=20250816T222437
[2025-08-16 22:24:37,527] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-16 22:24:37,581] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
